frontend/src/components/common/Resource/EditorDialog.tsx

```diff
@@ -62,11 +71,22 @@ export interface EditorDialogProps extends DialogProps {
   errorMessage?: string;
   /** The dialog title. */
   title?: string;
+  /** The flag for dispatching from a different component. */
+  externalDispatch?: boolean;
```

Comment:
Maybe a more direct name is better: `applyOnSave`.

Reply:
Changed the name and documented this